velox/docs/functions/spark/binary.rst

```diff
@@ -13,3 +13,7 @@ Binary Functions
 .. spark:function:: xxhash64(x) -> integer
 
     Computes the xxhash64 of x.
+
+.. spark:function:: sha1(x) -> string
```

Comment:
Functions in the documentation should appear in alphabetical order. Please, move accordingly.